src/rabbitmq/helm/README.md

```diff
@@ -0,0 +1,99 @@
+# RabbitMQ Plugin
+
+This is a resource plugin that provisions temporary vhosts in RabbitMQ
+for use within a Signadot Sandbox.
+
+## Installing the Plugin
+
+You will need a kubernetes cluster with the Signadot operator and RabbitMQ
+running the admin api (such as is found
+[here](https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq)).
+
+Before installing the plugin, create a Kubernetes Secret called `rabbitmq-auth` in
+the `signadot` namespace containing the 4 keys 
+- `svc-namespace`: the namespace in which the rabbit mq service exists.
+- `svc-name`: the name of the rabbitmq service.
+- `username`: the username with which to autheticate with `rabbitmqadmin`
+to create the vhost.
+- `password`: the password with which to autheticate with `rabbitmqadmin`
+to create the vhost.
+
+## Setting up the username and password in RabbitMQ
+
+The following example sets up a username and password for authenticating
+`rabbitmqadmin`
+
+```sh
+rabbitmqctl add_user test 123
+rabbitmqctl set_user_tags test administrator 
+rabbitmqctl set_permissions -p / test ".*" ".*" ".*"
+```
+
+The reader is referred to [the rabbitmq docs](https://www.rabbitmq.com/cli.html#remote-nodes)
+for more information.
+
+## Setting up the secret
+
+The following command is an example of setting up a secret as needed to install the
+plugin
+
+```sh
+kubectl -n signadot create secret generic rabbitmq-auth  \
+	--from-literal=svc-namespace=rabbit \
+	--from-literal=svc-name=rabbitmq \
+	--from-literal=username=test \
+	--from-literal=password=123
+```
+
+## Installing this plugin
+
+Make sure the target Kubernetes cluster already has Signadot Operator installed,
+and then install this chart with Helm:
+
+```sh
+helm repo add signadot-plugins-exp https://plugins.signadot.com/exp
+helm -n signadot install rabbitmq-plugin signadot-plugins-exp/rabbitmq
+```
+
+### Configuring the Plugin
+
+The following values can be configured for this chart via `--set` or `values.yaml`:
+
+Value | Description | Default
+----- | ----------- | -------
+`image` | The container image URL for the plugin | `signadot/rabbitmq-plugin:{version}`
+`imagePullPolicy` | The image pull policy for the plugin | `IfNotPresent`
+`rabbitAuthSecret` | The name of the Secret object in the `signadot` namespace containing the secrets described above. | 
+
+## Using the Plugin
+
+When creating a Signadot Sandbox, you can request a temporary rabbit mq vhost for
+an existing set of rabbitmq users from this plugin by specifying the plugin name 
+`sd-rabbitmq` and passing the following input parameters.
+
+Parameter | Description | Example
+--------- | ----------- | -------
+`refvhost` | The rabbitmq `vhost` to clone for users | `/`
```

Comment:
It seems like this param isn't used yet. Did you decide not to implement this for the first pass? If so, we should remove it from the docs.

Reply:
It is now implemented, at the cost of a problem building the image for linux/arm64, so it only pushes linux/amd64 now.

Reply:
I think we should fix arm before publishing since people are pretty likely to try these things in minikube on a mac.

I encountered some strange issues with apt when trying to cross-build for arm64, and it turned out I had missed the need to run this command from the [buildx docs](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images):

```sh
docker run --privileged --rm tonistiigi/binfmt --install all
```

Can you see if that helps for you, if you hadn't already run it?

Reply:
thanks, that did the trick.